driver/normalizer/tonode.go

```diff
@@ -15,4 +17,26 @@ var ToNode = &uast.ObjectToNode{
 	EndLineKey:         "loc.end.line",
 	ColumnKey:          "loc.start.column",
 	EndColumnKey:       "loc.end.column",
+	TokenKeys: map[string]bool{
+		"value": true,
+		"name":  true,
+	},
+	// Several maps with properties are returned by the AST, we should use only
+	// those with the key "type"
+	IsNode: func(v map[string]interface{}) bool {
+		_, ok := v["type"].(string)
+		return ok
+	},
+	// The offset position from babylon are based on 1, so we always sum one.
```

Comment:
Do you mean based on 0? BTW bblfsh offsets are 0-based, lines and columns (what I think you're referring here by "offset") are 1-based.